### ISSUE TITLE
Update user manual to 3.0.0

### ISSUE THREAD

**Issue by philip-cc** [cyber-crypt-com/encryptonize-core#214]
**User story**
As a ... (user role that this change targets)
I want ... (wanted behavior)
So that .. (expected benefit)

**Acceptance Criteria**
- ...

**Scope**
What is the scope of the change

**Not in Scope**
Anything that is not relevant for this change

**Non-functional requirements**
If any.. (performance, security, etc.)

**Additional context**

**Enabler stories**
- [ ] add any enabler story that should be done before starting this story to this list, and use the checkmark to track completion of enabler stories.
